File: reviews/src/main/java/com/ben/reviews/models/book/Book.java
package com.ben.reviews.models.book;

import com.ben.reviews.models.author.Author;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

import static jakarta.persistence.GenerationType.IDENTITY;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Entity(name = "book")
public class Book {
    @GeneratedValue(strategy = IDENTITY)
    @Id
    private Integer id;
    private String isbn;
    private String author;
    private String title;
    private String description;
    private String publication_date;
    private String cover_image;
    //TODO: cannot get this field populated
    @ManyToOne(/*fetch = FetchType.LAZY,*/ cascade = CascadeType.ALL)
    @JoinColumn(name = "author_id")
    private Author author_obj;
}

File: front/app/frontend/src/book.js

function Book(props) {
	console.log(props)
	const book = props.book
  return (
		<div style={{border:'1px solid black'}}>
		  <p>Title: {book.title}</p>
		  <p>Author: {book.author}</p>
		  <img src={book.cover_image} alt={"Cover"}></img>
		  <p>Description: {book.description || "----"}</p>
		  <p>ISBN: {book.isbn}</p>
		  <p>Publication Date: {book.publication_date}</p>
		</div>
	)
}

export default Book;
